packages/mobx-react/__tests__/observer.test.tsx

```diff
@@ -295,27 +296,6 @@ test("issue 12", () => {
     expect(events).toEqual(["table", "row: coffee", "row: tea", "table", "row: soup"])
 })
 
-test("changing state in render should fail", () => {
```

Comment:
This test is wrong on multiple levels:
I don't recall changing state in render should ever throw, with a slight exception of `autoAction`, which doesn't work ATM as mentioned elsewhere.
It doesn't `expect(...).toThrow()`, it just checks for the error (that is never thrown).
If it's about `enforceAction`, it should expect `console.warn` instead of an error.
It pollutes console with `Warning: Cannot update a component while rendering a different component` (this is a bit surprising given that only one component is involved, but not a concern atm).